jvm/library/src/main/kotlin/io/tira/tirex/tracker/Tracker.kt

```diff
@@ -671,11 +672,31 @@ class TrackingHandle private constructor(
     private fun export(result: Pointer) {
         if (exportFilePath == null) return
         when (exportFormat) {
-            null -> return
+            null -> exportGuessedFormat(result)
             ExportFormat.IR_METADATA -> exportIrMetadata(result)
         }
     }
 
+    private fun exportGuessedFormat(result: Pointer) {
```

Comment:
One small thing, should maybe .yml, .yaml, etc also be included and point to the irmetadata format?

Reply:
That could easily lead to conflicts once we support more metadata formats. In such cases, I'd rather have the user specify the format explicitly.